Few Migrations failing intermittently with - vCenter connection: failed to login

### What steps did you take and what happened?

This issue is observed 2/5 times while running migrations. Also, few of the migrations running and few failed.

<img width="1054" height="335" src="https://github.com/user-attachments/assets/5328d605-4377-444e-9a92-441b6ac48341" alt="Image">

Error logs -

```
2026/01/14 09:23:39 Failed to validate vCenter connection: failed to validate vCenter connection: failed to login: ServerFaultCode: Cannot complete login due to an incorrect user name or password.
2026/01/14 09:23:39 Connected to vCenter: vcenter.phx.pnap.platform9.horse

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x19ad3fb]

goroutine 1 [running]:
main.main()
        /home/runner/work/vjailbreak/vjailbreak/v2v-helper/main.go:90 +0x7fb
```

### What did you expect to happen?

Migrations should be successful

### Environment

Version: v0.3.8-8b5cc668

### vCenter version

NA

### Anything else you would like to add?

_No response_

--- a/v2v-helper/main.go
+++ b/v2v-helper/main.go
@@ -89,6 +89,7 @@ func main() {
 	vcclient, err := vcenter.VCenterClientBuilder(ctx, vCenterUserName, vCenterPassword, vCenterURL, vCenterInsecure)
 	if err != nil {
 		handleError(fmt.Sprintf("Failed to validate vCenter connection: %v", err))
+		return
 	}
 	utils.PrintLog(fmt.Sprintf("Connected to vCenter: %s\n", vCenterURL))
 	defer vcclient.VCClient.CloseIdleConnections()
@@ -96,6 +97,7 @@ func main() {
 	openstackclients, err := openstack.NewOpenStackClients(ctx, openstackInsecure)
 	if err != nil {
 		handleError(fmt.Sprintf("Failed to validate OpenStack connection: %v", err))
+		return
 	}
 	openstackclients.K8sClient = client
 	utils.PrintLog("Connected to OpenStack")
@@ -104,6 +106,7 @@ func main() {
 	thumbprint, err := vcenter.GetThumbprint(vCenterURL)
 	if err != nil {
 		handleError(fmt.Sprintf("Failed to get thumbprint: %s", err))
+		return
 	}
 	utils.PrintLog(fmt.Sprintf("VCenter Thumbprint: %s\n", thumbprint))
 
@@ -111,6 +114,7 @@ func main() {
 	vmops, err := vm.VMOpsBuilder(ctx, *vcclient, migrationparams.SourceVMName, client)
 	if err != nil {
 		handleError(fmt.Sprintf("Failed to get source VM: %v", err))
+		return
 	}
 	migrationobj := migrate.Migrate{
 		URL:                     vCenterURL,

